src/Codeception/Test/Loader.php

```diff
@@ -76,9 +81,32 @@ public function __construct(array $suiteSettings)
 
     public function getTests(): array
     {
+        if ($this->shard) {
+            $this->shard = trim($this->shard);
+            if (!preg_match('~^\d+\/\d+$~', $this->shard)) {
+                throw new ConfigurationException("Shard must be set as --shard=CURRENT/TOTAL where CURRENT and TOTAL are number. For instance: --shard=1/3");
+            }
+
+            list($shard, $totalShards) = explode('/', $this->shard);
```

Comment:
I think that we can use more modern `[$shard, $totalShards] = ` syntax here.

Reply:
I missed this feature of PHP 😁